arcxp/OBF-Starter
# Updated provisioning scripts to handle deployment

**If you have not filled out the checklist below, the pr is not ready for review.**

Updated provisioning scripts to handle deployment: FULL TESTING STEPS HERE: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/4109631661/Automated+Themes+and+OBF+Provisioning+Checklist

## Jira Ticket

- [THEMES-1711](https://arcpublishing.atlassian.net/browse/THEMES-1711)

## Acceptance Criteria


## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout deployment-scripts`
2. In your .env add the following values:
```
CONTENT_BASE=https://api.sandbox.{org}.arcpublishing.com 
ARC_ACCESS_TOKEN='{access token}'

OBF_SANDBOX_DEPLOYER_ENDPOINT=https://api.outboundfeeds-sandbox.{org}.arcpublishing.com 
OBF_SANDBOX_DEPLOYER_ACCESS_TOKEN='{access token}'
SANDBOX_RESIZER_ENCRYPTED='{resizer key}'

OBF_DEPLOYER_ENDPOINT=https://api.outboundfeeds-sandbox.{org}.arcpublishing.com 
OBF_DEPLOYER_ACCESS_TOKEN='{access token}'
PROD_RESIZER_ENCRYPTED='{resizer key}'
```
3. run `node obf-provisioning.js configure-and-deploy`, which will update, zip, deploy, and promote the bundle.
4. Inspect the zipped bundle, make sure all of the files look correct.
5. Make sure that the bundle has been successfully deployed to both obf environments
6. Inspect the obf environments to make sure that everything looks correct
